rom/dev/src/pcr.rs

```diff
@@ -91,12 +91,10 @@ pub(crate) fn extend_pcr0(
     pcr.extend(env.data_vault.fmc_tci(), PcrLogEntryId::FmcTci)?;
     pcr.extend_u8(env.data_vault.fmc_svn() as u8, PcrLogEntryId::FmcSvn)?;
     pcr.extend_u8(info.fmc.effective_fuse_svn as u8, PcrLogEntryId::FmcFuseSvn)?;
-    if let Some(vendor_lms_pub_key_idx) = info.vendor_lms_pub_key_idx {
-        pcr.extend_u8(
-            vendor_lms_pub_key_idx as u8,
-            PcrLogEntryId::LmsVendorPubKeyIndex,
-        )?;
-    }
+    pcr.extend_u8(
+        env.data_vault.lms_vendor_pk_index() as u8,
+        PcrLogEntryId::LmsVendorPubKeyIndex,
+    )?;
```

Comment:
We should log the status of ecc, or ecc+lms into the PCR.

Reply:
Updated